CHANGELOG.md

```diff
@@ -91,6 +93,51 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+### Optional changes
+
+We've recently made some other changes to GOV.UK Frontend. While these are not breaking changes, implementing them will make your service work better.
+
+You do not need to do anything if you're using Nunjucks macros.
+
+#### Change date input from `type=”number”` to `inputmode=numeric`
```

Comment:
```suggestion
#### Change date input from `type="number"` to `inputmode=numeric`
```

Reply:
Should we update the title to include `type="text"` too? Otherwise the title sounds a misleading

Reply:
Ah yes. Like this?
```suggestion
#### Change date input from `type=”number”` to `type="text"` and `inputmode=numeric`
```

Reply:
Yep, although we should probably also be consistent with the quotes, e.g: `type="text"` and `inputmode="numeric"`

Reply:
Definitely! Thanks.

```suggestion
#### Change date input from `type=”number”` to `inputmode="numeric"`
```